source/automated-deployment.rst

```diff
@@ -0,0 +1,84 @@
+Automating Deployment with Puppet
+=================================
+Definitions
+-----------
+* BEAMLINE_WORKSTATION
+    Any computer that beamline staff designate as a computer they want the
+    collection stack installed on
+* BEAMINE_SERVER
+    The main "compute" computer where jupyterhub will point to
+* IPYTHON_CONFIGURATION:
```

Comment:
What do you see as the downside of automated committing of all the python configuration files and adding a tag to note that this was the state of the beamline before the last puppetized update?